monkey/tests/monkey_island/stub_propagation_credentials_repository.py

```diff
@@ -0,0 +1,62 @@
+from typing import Sequence
+
+from monkey_island.cc.repository import ICredentialsRepository
+from monkey_island.cc.services.telemetry.processing.credentials import Credentials
```

Comment:
This should probably import `Credentials` from `common.credentials`.

Reply:
Yep but we don;t have serialization yet so I thought I will use the one in telemetry processing and then change it.

Reply:
I added TODOs to not forget it.